omnibus-sepolia-carina.toml

```diff
@@ -0,0 +1,539 @@
+
+[setting.min_margin_usd]
+defaultValue = "<%= parseEther('1000') %>"
+
+[setting.min_margin_ratio]
+defaultValue = "<%= parseEther('0.05') %>"
+
+[setting.incremental_margin_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.maintenance_margin_scalar]
+defaultValue = "<%= parseEther('0.5') %>"
+
+[setting.max_initial_margin_ratio]
+defaultValue = "<%= parseEther('0.9') %>"
+
+[setting.liquidation_reward_percent]
+defaultValue = "<%= parseEther('0.0001') %>"
+
+[setting.liquidation_limit_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.liquidation_max_pd]
+defaultValue = "<%= parseEther('0.0006') %>"
+
+[setting.liquidation_window_duration]
+defaultValue = "36"
+
+# --- Provision bfp market --- #
+#
+#
+
+[provision.bfp_market_factory]
+source = "<%= settings.bfp_market_package %>"
+targetPreset = "<%= settings.target_preset %>"
+options.coreProxyOwner = "<%= settings.owner %>"
+options.owner = "<%= settings.owner %>"
+options.salt = "<%= settings.salt %>"
+options.synthetixPackage = "<%= settings.snx_package %>@<%= settings.target_preset %>"
+options.synthetixPreset = "<%= settings.target_preset %>"
+depends = ["provision.system"]
+
+# --- Configure oracles for markets --- #
+#
+#
+
+# wstETH/stETH oracle
+[provision.wsteth_to_steth_ratio_oracle]
+source = "<%= settings.wsteth_to_steth_ratio_oracle %>"
+targetPreset = "<%= settings.target_preset %>"
+options.wsteth = "<%= settings.wsteth_address %>"
+
+[invoke.register_node_wsteth_to_steth_ratio_oracle]
+target = ["system.oracle_manager.Proxy"]
+func = "registerNode"
+args = [
+    # external node
+    2,
+    # contract
+    "<%= defaultAbiCoder.encode(['address'], [imports.wsteth_to_steth_ratio_oracle.contracts.WstEthToStEthRatioOracle.address]) %>",
+    [
+    ],
+]
+extra.wsteth_to_steth_ratio_oracle_id.event = "NodeRegistered"
+extra.wsteth_to_steth_ratio_oracle_id.arg = 0
+depends = ["provision.wsteth_to_steth_ratio_oracle"]
+
+# stETH/USD oracle
+#
+#
+#
+[invoke.register_node_steth_oracle_node]
+target = ["system.oracle_manager.Proxy"]
+func = "registerNode"
+args = [
+    # chainlink aggregator type
+    3,
+    # contract, twap, decimals
+    "<%= defaultAbiCoder.encode(['address', 'uint256', 'uint8'], [settings.steth_aggregator_address, 0, 8]) %>",
+    [
+    ],
+]
+extra.steth_oracle_node_id.event = "NodeRegistered"
+extra.steth_oracle_node_id.arg = 0
+depends = ["provision.system"]
+
+# wstETH/USD
+[invoke.register_node_wsteth_oracle_node]
+target = ["system.oracle_manager.Proxy"]
+func = "registerNode"
+args = [
+    # reducer node
+    1,
+    # mulDecimal
+    "<%= defaultAbiCoder.encode(['uint256'], [7]) %>",
+    # stETH/USD, wstETH/stETH
+    ["<%= settings.steth_oracle_node_id %>", "<%= settings.wsteth_to_steth_ratio_oracle_id %>"],
+]
+extra.wsteth_oracle_node_id.event = "NodeRegistered"
+extra.wsteth_oracle_node_id.arg = 0
+
+# --- Configure bfp --- #
+#
+#
+
+# --- Set pyth --- #
+#
+#
+#
+[invoke.set_pyth]
```

Comment:
As we will have this config part of global v3 config would be good to prefix invoke names accordingly.
Ideally these invoke names should not be needed at all, but as we still need them, my suggestion is to have names that are reflective of what they call (`ContractName_funcName`).

in this case would be 
```suggestion
[invoke.BfpMarketProxy_setPyth]
```
I have tried this approach in arbitrum deployments to avoid the need to figure out names.

Reply:
You want this prefex on all invokes that have target `bfp_market_factory.BfpMarketProxy`? Did you change `setPyth` to camelcase on purpose?

Reply:
I found this to be probably the easiest approach to name invokes. and add suffix for significant part if there are multiple invokes of same function.
this name should have been automatic, but as we have to write it manually - it is very annoying to "create" new names for every step, so the easiest approach I found is jut to blindly copypaste `Contract_funcName_maybeArgValue`. That also comes quite nicely in cannon UI and logs.

Reply:
<img width="796" alt="image" src="https://github.com/Synthetixio/synthetix-deployments/assets/28145325/58accb0c-704a-4264-8f44-0fc5024f08af">


see example here https://usecannon.com/packages/synthetix-omnibus/latest/421614-main